desc/geometry/core.py

```diff
@@ -111,16 +111,18 @@ def compute(
         ]
         calc0d = bool(len(dep0d))
         # see if the grid we're already using will work for desired qtys
-        if calc0d and (grid.N >= 2 * N + 5) and isinstance(grid, LinearGrid):
+        if calc0d and (grid.N >= 2 * self.N + 5) and isinstance(grid, LinearGrid):
             calc0d = False
 
         if calc0d and override_grid:
-            grid0d = LinearGrid(N=2 * N + 5, NFP=NFP, endpoint=True)
+            grid0d = LinearGrid(N=2 * self.N + 5, NFP=NFP, endpoint=True)
```

Comment:
is this grid creation jitable?

Reply:
yeah, self.N and NFP are both static attributes so are compile time constants.